src/lib/components/autocompleteAddress/index.tsx

```diff
@@ -250,15 +251,15 @@ const AutocompleteAddress = ({
           [styles['map-container--hidden']]: place === null,
         })}
       >
-        <div className={styles.map} id="map" />
+        <div className={styles.map} id="map" aria-hidden="true" />
```

Comment:
This is also a more general question: if we're hiding the loading status visual cue to assistive technologies, shall we find another way to communicate it ([like in this example](https://carbondesignsystem.com/components/loading/accessibility/#development-considerations) or in some of the changes you made in this PR)?

Reply:
This is just the map preview which is just visual and doesn't provide much value for screen readers